tests/markets/seer/test_price_manager.py

```
@@ -0,0 +1,159 @@
+
+    # Mock price data
+    with patch.object(PriceManager, "get_price_for_token") as mock_get_price:
+        mock_get_price.side_effect = [
+            CollateralToken(0.7),
+            CollateralToken(0.3),
+            CollateralToken(0.0),
+        ]  # Yes, No, Invalid probabilities
+        probability_map = price_manager.build_probability_map()
+
+    # Verify all keys are stripped and lowercase
+    assert all(key == key.strip().lower() for key in probability_map.keys())
```

Ruff (0.12.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)